lib/elixir/lib/option_parser.ex

```diff
@@ -492,4 +556,34 @@ defmodule OptionParser do
       :unknown
     end
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
+    total = Enum.count(errors)
+    error_pluralize = pluralize(total, "error", "errors")
```

Comment:
I feel like this may be simpler with just an inline `if` here, i.e.

```elixir
error = if total == 1, do: "error", else: "errors"
```

without the need for an external function with a *very* general name like `pluralize`. Wdyt?